sheetpr.go

```diff
@@ -33,14 +33,42 @@ type (
 	Published bool
 	// FitToPage is a SheetPrOption
 	FitToPage bool
-	// TabColor is a SheetPrOption
-	TabColor string
+	// TabColorIndexed is a TabColor option, within SheetPrOption
+	TabColorIndexed int
+	// TabColorRGB is a TabColor option, within SheetPrOption
+	TabColorRGB string
+	// TabColorTheme is a TabColor option, within SheetPrOption
+	TabColorTheme int
+	// TabColorTint is a TabColor option, within SheetPrOption
+	TabColorTint float64
+	// Equivalent to TabColorRGB, alias added for backwards compatibility
+	TabColor = TabColorRGB
```

Comment:
Remove the `TabColor`, using the `TabColorRGB` instead of it, I will add a notice of this breaking change in the release note of [the next released version](https://xuri.me/excelize/en/releases/v2.6.1.html).

Reply:
updated